core/src/main/resources/hudson/model/Job/configure.jelly

```diff
@@ -26,50 +26,64 @@ THE SOFTWARE.
   Config page. derived class specific entries should go to configure-entries.jsp
 -->
 <?jelly escape-by-default='true'?>
-<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form">
-
-  <l:layout type="one-column" permission="${it.EXTENDED_READ}" title="${%Config(it.displayName)}">
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:l="/lib/layout" xmlns:f="/lib/form" xmlns:p="/lib/hudson/project">
+  <l:layout permission="${it.EXTENDED_READ}" title="${%Config(it.displayName)}">
     <j:set var="readOnlyMode" value="${!it.hasPermission(it.CONFIGURE)}" />
 
-    <l:js src="jsbundles/config-scrollspy.js" />
-    <l:css src="jsbundles/config-scrollspy.css" />
+    <l:side-panel>
+      <div id="tasks">
+        <div class="jenkins-app-bar">
+          <div class="jenkins-app-bar__content">
+            <h1>${%Configuration}</h1>
+          </div>
+        </div>
+      </div>
+    </l:side-panel>
 
     <l:main-panel>
-    <div class="container">
-      <div class="row">
-        <div class="jenkins-config-container">
+      <script src="${resURL}/jsbundles/section-to-sidebar-items.js" type="text/javascript"/>
+
+      <div class="behavior-loading"><l:spinner text="${%LOADING}"/></div>
 
-          <div class="behavior-loading"><l:spinner text="${%LOADING}"/></div>
+      <f:form method="post" class="jenkins-form" action="configSubmit" name="config" tableClass="config-table">
+        <div class="jenkins-app-bar">
+          <div class="jenkins-app-bar__content">
+            <h2>
+              <div class="jenkins-hidden">
+                <l:icon src="symbol-settings" />
+              </div>
+              <span>${%General}</span>
+            </h2>
+          </div>
+          <div class="jenkins-app-bar__controls">
+            <p:config-disableBuild />
+          </div>
+        </div>
 
-          <f:form method="post" action="configSubmit" name="config" tableClass="config-table scrollspy">
-            <j:set var="descriptor" value="${it.descriptor}" />
-            <j:set var="instance" value="${it}" />
+        <j:set var="descriptor" value="${it.descriptor}" />
+        <j:set var="instance" value="${it}" />
 
-            <div class="jenkins-section">
-              <f:entry title="${%Description}" help="${app.markupFormatter.helpUrl}">
-                <f:textarea name="description" value="${it.description}" codemirror-mode="${app.markupFormatter.codeMirrorMode}" codemirror-config="${app.markupFormatter.codeMirrorConfig}" previewEndpoint="/markupFormatter/previewDescription"/>
-              </f:entry>
-            </div>
+        <div class="jenkins-section jenkins-section--no-border" style="margin-top: 2.5rem;">
```

Comment:
Is there a css class we could use instead of `style`?

Reply:
Done 👍